demo/config/services.yaml

```diff
@@ -19,10 +19,10 @@ services:
             - '../src/Entity/'
             - '../src/Kernel.php'
 
-    Symfony\AI\McpSdk\Capability\Tool\ToolExecutorInterface:
+    mcp.tool_executor:
```

Comment:
Not quite sure about this.

I'm not familiar with this MCP bundle at all – is this really how this is supposed to work? Don't we have some autoconfiguration?

Reply:
See #168 :)